packages/generator/src/v2.0/UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst.ts

```diff
@@ -126,43 +126,15 @@ export function __UNSTABLE_wireInterfaceTypeV2ToSdkObjectConst(
   };
 
   function getV2Types() {
-    return `import type {
-      AggregateOpts as $AggregateOpts,
-      AggregateOptsThatErrorsAndDisallowsOrderingWithMultipleGroupBy as $AggregateOptsThatErrorsAndDisallowsOrderingWithMultipleGroupBy,
```

Comment:
Remove a ton of types that weren't being used but were being emitted (and cleaned up  by organize imports)